mkdocs.yml

```diff
@@ -24,14 +24,16 @@ extra:
       short_name: "CC BY-SA"
       url: "https://creativecommons.org/licenses/by-sa/4.0/"
     cdpa:
+      date: 2023-10-02
       name: "Data Processing Agreement"
-      url: "https://s3-kna1.citycloud.com/6a5aa55d8f094a13ae18199639aa72c2:cleura.files/City_Network_CDPA.pdf"
+      url: "https://s3-kna1.citycloud.com/6a5aa55d8f094a13ae18199639aa72c2:cleura.files/Cleura_Data_Processing_Agreement_for_Public_Cloud-V23.10.2.pdf"
     termination_form:
       name: "Termination of Subscription Form"
       url: "https://s3-kna1.citycloud.com/6a5aa55d8f094a13ae18199639aa72c2:cleura.files/Cleura-Form-Termination-Of-Subscription.pdf"
     tos:
+      date: 2023-09-08
```

Comment:
@armanborghem: Here, too.

Reply:
I'm also not sure what this date refers to. Is it from this page?
https://docs.cleura.cloud/howto/account-billing/delete-account/#using-the-termination-of-subscription-form
We haven't made any changes to the termination of subscription form, so I'm not sure why we would change a date there...

Reply:
The `date` field here refers to the effective date of the ToS. It is mentioned here: <https://docs.cleura.cloud/reference/legal/#terms-of-service>

Reply:
The document was published 2023-10-03 so that's the date it should say that it's applicable from. But I would prefer if the text was changed to:

Our standard Terms of Service for Public Cloud were last updated on 2023-10-03.
(link "Terms of Service for Public Cloud")